### DISCUSSION TITLE
鳝

### DISCUSSION THREAD

**Discussion by dingzhen-vape** [share121/inter-knot#1088] [其他]
![因为他鳝](https://github.com/user-attachments/assets/74807df7-457a-417d-83bd-83ffb87a0e57)

**Comment by match-fire** [comment]
你的丁真抽瑞克5代还是维普5代？
小心我拿维普6代吊打你

**Comment by inkluv** [comment]
问下楼上，vape有v5了？我是不是断网了

**Comment by su2569** [comment]
![326D861136F9C53F1FB744C9CB9C6447](https://github.com/user-attachments/assets/b38215f8-678a-4f6c-b7b2-f8761e2e935b)

**Reply by su2569** [reply]
因为他鳝（善）
